### ISSUE TITLE
Sticky footer not working

### ISSUE THREAD

**Issue by annirak** [diprokon/ng-table-virtual-scroll#14]
Hi, I have a table with sticky header & footer.
The header is working fine, but the footer stops sticking after a couple of scrolls

**Comment by anissaba** [comment]
Hello,same problem.

**Comment by blogcraft** [comment]
Same Here.

**Comment by ahimik** [comment]
+1.
I just started using virtual scrolling for the first time and immediately raised the same issue.
I could be wrong since I spend just a few minutes looking into the code, but my impression that it can be fixed very easily, all you need is to change the following piece of code:
```
setSticky(offset) {
    this.scrollStrategy.viewport.elementRef.nativeElement.querySelectorAll(stickySelector)
      .forEach((el: HTMLElement) => {
        const parent = el.parentElement;
        let baseOffset = 0;
        if (this.stickyPositions.has(parent)) {
          baseOffset = this.stickyPositions.get(parent);
        }
        el.style.top = `-${-baseOffset + offset}px`;
      });
  }
```
, since both footer and header contains the same stickySelector both of them get negative top offset while footer need positive bottom offset instead, so simply splitting stickySelector into two separate selectors: `const stickyHeaderSelector= '.mat-header-row.mat-table-sticky'` and `const stickyFooterSelector='.mat-footer-row.mat-table-sticky'` and then applying for header:
`el.style.top = -${-baseOffset + offset}px;`
and for footer:
`el.style.bottom = ${-baseOffset + offset}px;`
should fix the issue.

**Comment by diprokon** [comment]
Fixed in 1.3.3
